packages/massa-web3/src/experimental/basicElements/address.ts

```diff
@@ -140,6 +155,6 @@ export class Address {
   toString(): string {
     return `${ADDRESS_PREFIX}${
       this.isEOA ? ADDRESS_USER_PREFIX : ADDRESS_CONTRACT_PREFIX
-    }${this.serializer.serialize(this.versionedBytes())}`
+    }${this.serializer.serialize(this.versionedBytes().slice(1))}`
```

Comment:
This is unclear but I don't find ways to improve this right now in my head.

Reply:
The source of the problem is the choice made at the node level to use this encoding:
- The base58 encoded bytes contain the version, but not the type of the address.
- During signature, the type of the address must be used (and I don't see any reason to do so).

So, we ended up with a bicephalous solution matching the bicephalous conception...

Reply:
yeah it's not that important